src/models/config/ChatAnalyticsConfig.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * The main configuration options for Chat Analytics.
+ *
+ * @public
+ */
+export default interface ChatAnalyticsConfig {
+  /** The API key for accessing the Analytics Events API. */
+  apiKey: string; // TODO: Add optional "token" for JWT auth
```

Comment:
should we make an item for this TODO so we don't forget?

Reply:
Sure I can open up a GH issue after this PR is merged.